File: ESTUDO DE LINGUAGENS/PYTHON/Questões Uri/1555.py
casos=int(input())

for cont in range(0,casos,1):
  x,y=map(int, input().split(" "))
  rafa=(((3*x)**2)+(y**2))
  beto=((2*(x**2))+((5*y)**2))
  carlos=((-100*x)+(y**3))
  if rafa>beto and rafa>carlos:
    print("Rafael ganhou")
  if beto>rafa and beto>carlos:
    print("Beto ganhou")
  if carlos>rafa and carlos>beto:
    print("Carlos ganhou")

File: ESTUDO DE LINGUAGENS/JAVASCRIPT/Questões uri/1555.js
const lista=(line)=>line.split(" ").map(a => +a)
let qtd=+lines.shift()
for(let cont=0; cont<qtd; cont++){
    let[x,y]=lista(lines.shift())
    let  rafa=(((3*x)**2)+y**2)
    let beto=(2*(x**2)+((5*y)**2))
    let carlo=((-100*x)+(y**3))
    if (rafa>beto && rafa>carlo){
        console.log("Rafael ganhou")
    }
    else if(beto>rafa && beto>carlo){
        console.log("Beto ganhou")
    }
    else if(carlo>rafa && carlo>beto){
        console.log("Carlos ganhou")
    }
}